perf/benchmark/run_benchmark_job.sh

```diff
@@ -0,0 +1,101 @@
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+ROOT=$(dirname "$WD")
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+# This is config for postsubmit cluster.
+export VALUES="${VALUES:-values-istio-postsubmit.yaml}"
+export DNS_DOMAIN="fake-dns.org"
+export FORTIO_CLIENT_URL=""
+
+function setup_metrics() {
+  # shellcheck disable=SC2155
+  export FORTIO_CLIENT_URL=http://$(kubectl get services -n twopods fortioclient -o jsonpath="{.status.loadBalancer.ingress[0].ip}"):8080
+  if [[ -z "$FORTIO_CLIENT_URL" ]];then
+    kubectl -n twopods port-forward svc/fortioclient 8080:8080 &
+    export FORTIO_CLIENT_URL=http://localhost:8080
+  fi
+  export PROMETHEUS_URL=http://localhost:9090
+  kubectl -n istio-prometheus port-forward svc/istio-prometheus 9090:9090 &>/dev/null &
+}
+
+function collect_metrics() {
+  local GENERATE_GRAPH=$1
+  local PLOT_METRIC=$2
+  CSV_OUTPUT="$(mktemp /tmp/benchmark_XXXX.csv)"
+  pipenv install
+  pipenv run python fortio.py $FORTIO_CLIENT_URL --csv_output="$CSV_OUTPUT" --prometheus=$PROMETHEUS_URL \
+   --csv StartTime,ActualDuration,Labels,NumThreads,ActualQPS,p50,p90,p99,cpu_mili_avg_telemetry_mixer,cpu_mili_max_telemetry_mixer,\
+mem_MB_max_telemetry_mixer,cpu_mili_avg_fortioserver_deployment_proxy,cpu_mili_max_fortioserver_deployment_proxy,\
+mem_MB_max_fortioserver_deployment_proxy,cpu_mili_avg_ingressgateway_proxy,cpu_mili_max_ingressgateway_proxy,mem_MB_max_ingressgateway_proxy
+
+  if [[ "$GENERATE_GRAPH" = true ]];then
+    BENCHMARK_GRAPH="$(mktemp /tmp/benchmark_graph_XXXX.html)"
+    pipenv run python graph.py "${CSV_OUTPUT}" "${PLOT_METRIC}" --charts_output="${BENCHMARK_GRAPH}"
+    dt=$(date +'%Y%m%d-%H')
+    RELEASE="$(cut -d'/' -f3 <<<"${CB_GCS_FULL_STAGING_PATH}")"
+    GRAPH_NAME="${RELEASE}.${dt}.${PLOT_METRIC}"
+    gsutil -q cp "${BENCHMARK_GRAPH}" "gs://$CB_GCS_BUILD_PATH/${GRAPH_NAME}"
+  fi
+}
+
+echo "Setup istio release: $TAG"
+pushd "${ROOT}/istio-install"
+   ./setup_istio.sh "${TAG}"
+popd
+# install dependencies
+cd "${WD}/runner"
+pipenv install
+# setup test
+pushd "${WD}"
+./setup_test.sh
+popd
+
+setup_metrics
+echo "Start running perf benchmark test."
+# For adding or modifying configurations, refer to perf/benchmark/README.md
+# Configuration1
+EXTRA_ARGS="--serversidecar --baseline"
+CONN=16
+QPS=500,1000,1500,2000
+DURATION=300
+METRIC="cpu"
+# shellcheck disable=SC2086
+pipenv run python runner.py ${CONN} ${QPS} ${DURATION} ${EXTRA_ARGS}
+collect_metrics true ${METRIC}
+METRIC="mem"
+collect_metrics true ${METRIC}
+
+# Configuration2
+CONN=4,8,16,32,64
+QPS=500
```

Comment:
it is better to follow the other test conventions to add `1` and `2` connections here.

Reply:
Also, could we set QPS to be 1000? since our published perf latency results is sending 1000 qps.

Reply:
@carolynhu @mandarjog done, thx